test/integration/util/util.go

```diff
@@ -92,15 +94,36 @@ func NewKubectlRunner(t *testing.T) *KubectlRunner {
 }
 
 func (k *KubectlRunner) RunCommand(args []string, outputObj interface{}) error {
```

Comment:
I'd rename this to "RunCommandParseOutput", and rename RunCommandSimple to "RunCommand".